lib/migrate/index.js

```diff
@@ -48,7 +48,7 @@ function transformSingleAST(ast, source, transformFunction) {
  * @param { String } source - Source file contents
  * @param { Array<Function> } transformations - List of trnasformation functions in defined the
  * order to apply. By default all defined transfomations.
- * @param { Object } options - Reacst formatting options
+ * @param { Object } options - Recast formatting options
```

Comment:
Could you add this JSdoc too?

Reply:
Sure, no problem. Do you mean the jsdoc for the properties of the `options` object?

Reply:
No, you forgot to do this/add the extra *. And `Array<Function>` is wrong, should be `Function[]`